integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/TestQueryWithColumnMetCacheAndCacheLevelProperty.scala

```diff
@@ -367,5 +367,31 @@ class TestQueryWithColumnMetCacheAndCacheLevelProperty extends QueryTest with Be
     sql("DROP table IF EXISTS carbonCahe")
   }
 
+  // TODO: support insert and query with filter
+  ignore("Test For Cache set but Min/Max exceeds for Binary") {
```

Comment:
As discussed for above COLUMN_META_CACHE, no need to support for binary columns.

Reply:
removed.